wuvt/donate/models.py

```diff
@@ -70,3 +70,11 @@ def set_paid(self, method, paid_date=None):
             self.paid_date = paid_date
         else:
             self.paid_date = datetime.datetime.utcnow()
+
+    def set_shipped(self, shipped_date=None):
+        if self.premiums is not None:
```

Comment:
So if premiums is `None`, then this function does nothing. Might be a good idea to instead return an error.